modules/configure_project/main.tf

```diff
@@ -100,3 +100,30 @@ module "watsonx_ai_runtime_crn_parser" {
   version = "1.1.0"
   crn     = var.watsonx_ai_runtime_crn
 }
+
+data "ibm_iam_auth_token" "tokendata" {}
+
+locals {
+  sensitive_tokendata = sensitive(data.ibm_iam_auth_token.tokendata.iam_access_token)
+}
+
+resource "null_resource" "add_collaborators_to_project" {
+  for_each   = { for member in var.watsonx_ai_new_project_members : member.email => member }
+  depends_on = [data.ibm_iam_auth_token.tokendata]
+  triggers = {
+    always_run = timestamp()
```

Comment:
@kierramarie this is why consistency test is failing. Why do you need to always run this? Wouldn't it only need to be run if he value of `var.watsonx_ai_new_project_members` changed?